ros2-launchs/ros2-lidar-odometry.launch.py

```
@@ -1,19 +1,49 @@
-from launch.actions import DeclareLaunchArgument
-from launch.actions import SetEnvironmentVariable
-from launch.actions import GroupAction
-from launch.actions import Shutdown
+from launch_ros.actions import Node, PushRosNamespace
+from launch.actions import (DeclareLaunchArgument, SetEnvironmentVariable,
+                            GroupAction, Shutdown, OpaqueFunction)
 from ament_index_python import get_package_share_directory
 import os
 
 
+def resolve_state_estimator_config(context, *args, **kwargs):
```

Ruff (0.14.14)

Unused function argument: `args`

(ARG001)